src/main/resources/application.properties

```diff
@@ -43,7 +43,7 @@ spring.jpa.show-sql=false
 spring.main.banner_mode=off
 
 
-pa.scheduler.url=http://localhost:8080
+pa.scheduler.url=http://trydev.activeeon.com:8080
```

Comment:
It seems you didn't commit that intentionally: default value should be `http://localhost:8080`